packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,78 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  opacity: 0.6;
+}
+
+.spinner:after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: var(--space-larger);
+  height: var(--space-larger);
+  box-shadow: var(--shadow-dark);
+  margin: -1.125rem 0 0 -1.125rem;
```

Comment:
These margins are using magic `rems` while width and height are using the variable `--space-larger`.

A more safe margin might be something like:

```css
margin-top: calc(var(--space-larger) / 2 * -1);
margin-left: calc(var(--space-larger) / 2 * -1);
```

This way if the var changes we are still safe.

Reply:
Totally, that's way better! Fixed!